quarkus/pom.xml

```diff
@@ -31,8 +31,10 @@
     <packaging>pom</packaging>
 
     <properties>
-        <quarkus.version>1.0.1.Final</quarkus.version>
-        <resteasy.version>4.3.1.Final</resteasy.version>
+        <quarkus.version>1.2.0.Final</quarkus.version>
+        <resteasy.version>4.4.2.Final</resteasy.version>
+        <jackson.version>2.10.0</jackson.version>
```

Comment:
Quarkus 1.2.0 uses 2.10.2 (https://github.com/quarkusio/quarkus/blob/1.2.0.Final/bom/runtime/pom.xml#L79)

Reply:
thanks for pointing out, fixed